packages/metals-languageclient/src/getJavaHome.ts

```diff
@@ -53,27 +53,55 @@ async function validateJavaVersion(
   return false;
 }
 
+function propertyValueOf(
+  input: string,
+  propertyName: string
+): string | undefined {
```

Comment:
Could we wrap it in a try just in case?

Reply:
Which part specifically?

Reply:
I would say the whole method. I think it's pretty safe, but every string manipulation bites us at some point however good it looks :sweat_smile:

Reply:
Hm, if the output of the java process would be broken somehow we could end up trying to grab the second part of a split on `=`, where there is no second part. This would raise an index out of bounds-exception.

I'll add a `try/catch() { log(…); return }` statement around the split-part then :)

Reply:
Thanks!